cli/module_graph.rs

```diff
@@ -824,7 +829,11 @@ impl Graph {
       }));
     }
     let maybe_ignored_options =
-      config.merge_tsconfig(options.maybe_config_path)?;
+      if let Some(config_file) = options.maybe_config_file.as_ref() {
```

Comment:
I'm ok with this, but if we have `ConfigFile::as_compiler_options()` we could in theory simply this?  Also, wouldn't it be cleaner to have whatever method accept an `Option<ConfigFile>` so you have only only deal with the option unwrapping once instead of several places?

Reply:
Valid point about `Option<ConfigFile>`, but if I'm add `ConfigFile::as_compiler_options` it will still required to do the unwrap. To be honest I don't really mind that - the code is still readable, it's just there are additional 3 lines.